app/models/lockstep/user.rb

```diff
@@ -3,17 +3,25 @@ class Lockstep::User < Lockstep::ApiRecord
   self.id_ref = "user_id"
   load_schema(Schema::UserAccount)
 
+  # # Load the first account
+  belongs_to :account, class_name: "Lockstep::Account", loader: ->(user) { Lockstep::Account.first }
+
+  alias_attribute :last_logged_in, :b2_c_last_logged_in
+
   def self.invite(email_addresses)
     emails = email_addresses.map { |email| { email: email } }
     resp = resource.post('/invite', body: emails, params: {})
     parsed_response = JSON.parse(resp.body)
-    raise Lockstep::Exceptions::BadRequestError, parsed_response['title'] unless resp.code == '200'
+    if resp.code != '200'
+      if resp.code == '401'
+        raise Lockstep::Exceptions::UnauthorizedError, 'Unauthorized: Check your App ID & Master Key'
```

Comment:
@sumitsourav This hardcoding seems incorrect. I think we should relay the error message that comes from the Platform

Reply:
@mandarvaze  i implemented the error scenarios same as `ApiRecord#bulk_import`, suggested by vivek.

Reply:
Resolved [here](https://github.com/Lockstep-Network/lockstep-sdk-ruby-rails/pull/44/commits/f209974216f0741f9e50f94aedd274fe82300a39)

Reply:
@mandarvaze This is the implementation in `ApiRecord` all all API calls at the moment. I don't recollect why did we hard code it, might be that we didn't get an appropriate message from the platform.